### PR TITLE
Add subcategory in docs

### PR DESCRIPTION
**Pull request by danielmkn**
Subcategory organizes documents in the Terraform Registry UI

### CODE REVIEW

**Review by chb0github — APPROVED** [review]

File: docs/resources/license_policy.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 page_title: "xray_license_policy Resource - terraform-provider-xray"
-subcategory: ""
-description: |-
-  Creates an Xray policy using V2 of the underlying APIs. Please note: It's only compatible with Bearer token auth method (Identity and Access => Access Tokens
+subcategory: "Policies"
 ---
 
-# xray_license_policy (Resource)
+Creates an Xray Policy using V2 of the underlying APIs.
+Please note: It's only compatible with Bearer token auth method (Identity and Access => Access Tokens.
```
Comment (alexhung) [inline docs/resources/license_policy.md:8]: Missing a closing parenthesis.

File: docs/resources/security_policy.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 page_title: "xray_security_policy Resource - terraform-provider-xray"
-subcategory: ""
-description: |-
-  Creates an Xray policy using V2 of the underlying APIs. Please note: It's only compatible with Bearer token auth method (Identity and Access => Access Tokens
+subcategory: "Policies"
 ---
 
-# xray_security_policy (Resource)
+Creates an Xray Policy using V2 of the underlying APIs.
+Please note: It's only compatible with Bearer token auth method (Identity and Access => Access Tokens.
```
Comment (alexhung) [inline docs/resources/security_policy.md:8]: Same here

**Review by alexhung — CHANGES_REQUESTED** [review]
LGTM except the missing closing parenthesis

File: docs/resources/license_policy.md (hunk 1 above)
Comment (danielmkn) [reply docs/resources/license_policy.md:8]: thanks! Fixed!

File: docs/resources/security_policy.md (hunk 2 above)
Comment (danielmkn) [reply docs/resources/security_policy.md:8]: thanks! Fixed!

**Review by alexhung — APPROVED** [review]

**Review by maheshjfrog — APPROVED** [review]

**Pull request merged**